api/src/display.rs

```diff
@@ -5,30 +5,31 @@
 
 use crate::printable;
 use crc::crc32;
-use failure::Fail;
+use displaydoc::Display;
 use protobuf::{parse_from_bytes, Message};
 
 /// Decoding / encoding errors
-#[derive(Clone, Debug, Eq, PartialEq, Fail)]
+#[derive(Clone, Debug, Eq, PartialEq, Display)]
 pub enum Error {
     /// Protocol buffer could not be serialized
```

Comment:
The comment should include `{0}` so that displaydoc properly includes the extra data (e.g. `Serialization error: {0}`)